controller/src/services/credential/credential.service.ts

```diff
@@ -0,0 +1,27 @@
+// Initializes the `credential` service on path `/issuer/credential`
+import { ServiceAddons } from '@feathersjs/feathers';
+import { Application } from '../../declarations';
+import { Credential } from './credential.class';
+import hooks from './credential.hooks';
+
+// Add this service to the service type index
+declare module '../../declarations' {
+  interface ServiceTypes {
+    'issuer/credential': Credential & ServiceAddons<any>;
+    // 'issuer/credential/send': CredentialSend & ServiceAddons<any>;
+  }
+}
+
+export default function (app: Application): void {
+  const options = {
+    paginate: app.get('paginate')
+  };
+
+  // Initialize our service with any options it requires
+  app.use('/issuer/credential', new Credential(options, app));
+  // app.use('/issuer/credential/send', new CredentialSend(options, app));
```

Comment:
Unless there is a good reason to keep these mappings, I would remove the commented code for cleanliness.

Reply:
Removed unused endpoints and code